### PR TITLE
docs: fix some problem

### PR DESCRIPTION
**Pull request by MachaelLee**
fix some problem

### CODE REVIEW

File: docs/src/en/sdk/java.md [hunk 1]
```diff
@@ -36,6 +36,15 @@ if (!client.init(opts)) {
 }
 ```
 
+The initialization requires at least three parameters:
+
+- `Endpoint`: 127.0.0.1
+- `Port`: 8831
+- `RouteMode`: DIRECT/PROXY
+
+`Endpoihnt` and `Port` are simple.
```
[the remark this answers is not in the document]
Comment (mrrtree) [reply docs/src/en/sdk/java.md:45]: `Endpoihnt` and `Port` are simple.

This is very strange, why don't you just delete it

[the remark this answers is not in the document]
Comment (MachaelLee) [reply docs/src/en/sdk/java.md:45]: done

**Review by mrrtree — APPROVED** [review]
LGTM

**Review by Rachelint — APPROVED** [review]
LGTM

**Pull request merged**